### PR TITLE
bugfix/image-build-resource-not-found

### PR DESCRIPTION
**Pull request by Anonycoders**
- PENDING check statement added.
- counter set to 45 at max
- sleep duration set to 4 sec
- cli will wait 3 min to receive logs or will get out of the loop.

### CODE REVIEW

File: fandogh_cli/image_commands.py [hunk 1]
```diff
@@ -62,13 +62,22 @@ def list_images():
...
 def show_image_logs(image_name, version):
     image_offset = 0
+    counter = 0
 
     if not image_name:
         image_name = get_project_config().get('image.name')
     while True:
         response = get_image_build(image_name, version, image_offset)
+
+        if response.get('state') == 'PENDING' and \
```
Comment (s-soroosh) [inline fandogh_cli/image_commands.py]: why is this check is needed?

File: fandogh_cli/image_commands.py [hunk 2]
```diff
@@ -62,13 +62,22 @@ def list_images():
...
+            counter += 1
+            sleep(4)
+            continue
+
         image_offset = response.get('lines_count')
         logs = response.get('logs')
+
         if logs.strip():
             click.echo(response.get('logs'))
         if response.get('state') != 'BUILDING':
```
Comment (psycho-ir) [inline fandogh_cli/image_commands.py]: you probably just need to update this line of code to check set is not bulilding or pending.

File: fandogh_cli/image_commands.py (hunk 1 above)
Comment (s-soroosh) [reply fandogh_cli/image_commands.py]: you probably just need to update line 83 to check state is not bulilding or pending.

Comment (Anonycoders) [reply fandogh_cli/image_commands.py]: Good point

**Review by s-soroosh — APPROVED** [review]

**Pull request merged**